src/codewhisperer/service/recommendationHandler.ts

```diff
@@ -136,13 +136,15 @@ export class RecommendationHandler {
     }
 
     async getTaskTypeFromEditorFileName(fileName: string): Promise<CodewhispererGettingStartedTask> {
-        if (fileName.startsWith('CodeWhisperer_generate_suggestion')) {
+        const parts = fileName.split('/')
```

Comment:
does this work on windows path?

Reply:
You'll want to use `path.basename()` instead

Reply:
I used 
` if (filePath.includes('CodeWhisperer_generate_suggestion'))`

I think this should work for both